src/components/Input.js

```diff
@@ -4,13 +4,23 @@ export const Input = styled.input `
   border-color: ${({theme}) => theme.colors.grey};
   padding: ${({theme}) => `${theme.space[1]}px`};
   width: 100%;
+  height: 35px;
   color: ${({theme}) => theme.colors.darkGrey};
+  background-color: ${({theme}) => theme.colors.lightGrey};
   border: none;
-  border-bottom: solid 1x ${({theme}) => theme.colors.blue};
   &:focus {
-    border: none;
     border: solid 2px ${({theme}) => theme.colors.blue};
     outline: none;
-    border-radius: ${({theme}) => `${theme.radii[1]}px`};
   }
 `
+export const Select = styled.select`
```

Comment:
You can extract some of these parts that are the same for `Input` and `Select` components using this function (https://styled-components.com/docs/api#css) and reuse it in both of them. What do you think?